src/test/java/org/fogbowcloud/manager/core/plugins/cloud/openstack/OpenStackAttachmentPluginTest.java

```diff
@@ -61,30 +61,36 @@ public void setUp() {
         this.client = Mockito.mock(HttpRequestClientUtil.class);
         this.openStackAttachmentPlugin.setClient(this.client);
     }
-
+    
+    //test case: Check if requestInstance is returning the instanceId from Json response properly.
     @Test
-    public void requestInstanceTest()
-            throws FogbowManagerException, HttpResponseException, UnexpectedException {
+    public void testRequestInstance() throws FogbowManagerException, HttpResponseException, UnexpectedException {
+    	//set up
         Mockito.doReturn(FAKE_POST_REQUEST_BODY).when(this.client).doPostRequest(
                 Mockito.anyString(), Mockito.any(Token.class), Mockito.any(JSONObject.class));
-
+        
+        //exercise
         String instanceId = this.openStackAttachmentPlugin.requestInstance(this.attachmentOrder,
                 this.localToken);
+        
+        //verify
         Assert.assertEquals(FAKE_INSTANCE_ID, instanceId);
     }
-
+    
+    //test case: Check if requestInstance throws UnexpectedException when the http requisition throws UnexpectedException.
     @Test(expected = UnexpectedException.class)
-    public void requestInstanceThrowUnexpectedExceptionTest()
+    public void testRequestInstanceThrowUnexpectedException()
```

Comment:
@ordansantos I suggest that you talk with the original author to understand the actual test case. After that, we can talk about it

Reply:
The test was properly rewritten